staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -828,8 +838,12 @@ func (meal *cfgMeal) finishQueueSetReconfigsLocked() {
 		if limited := plState.pl.Spec.Limited; limited != nil {
 			if qCfg := limited.LimitResponse.Queuing; qCfg != nil {
 				meal.maxWaitingRequests += int(qCfg.Queues * qCfg.QueueLengthLimit)
+
+				plState.maxSeats = uint64(math.Max(float64(1), math.Min(math.Ceil(float64(concurrencyLimit)*priorityLevelMaxSeatsPercent),
```

Comment:
Sorry for this comment, but I actually liked the
```
handSize := ...
```

before - it made thing more readable